huaweicloud/config/endpoints.go

```diff
@@ -473,6 +473,15 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Name:             "tms",
 		Version:          "v1.0",
 		Scope:            "global",
+		Admin:            true,
```

Comment:
it's better to add a comment about why we use admin

Reply:
done.